### ISSUE TITLE
Tabs don't switch with hotkeys sometimes

### ISSUE THREAD

**Issue by karevn** [zeit/hyper#1425]
Can't switch Hyper tabs with `Shift-Command-[` and `Shift-Command-]` hotkeys sometimes. It starts to work again after switching with mouse.

-

Hyper 1.1.0
Electron 1.4.7
darwin x64 16.3.0

**Comment by matheuss** [comment]
@karevn could you confirm if this happens only after you switch back from a new tab?
1. you have only one open tab
2. cmd+t
3. switch to the other tab
4. the hotkeys will *not* work (you will not be able to type anything)
5. you switch with the mouse to the other tab OR you click anywhere
6. the shortcuts works again

**Comment by karevn** [comment]
@matheuss yes, it works exactly like that.

**Comment by chabou** [comment]
Seems to be a duplicate of #1273
Fixed in master and will be shiped in next release

**Comment by matheuss** [comment]
@karevn then it's indeed a duplicate of #1273, @karevn. We'll ship it ASAP. Thanks for reporting 🙆
